Documentation/Troubleshooting/disaster-recovery.md

```diff
@@ -245,80 +245,132 @@ continue to ensure cluster health. Upgrades will be blocked, further updates to
 Since Kubernetes does not allow undeleting resources, the following procedure will allow you to restore
 the CRs to their prior state without even necessarily suffering cluster downtime.
 
-1. Scale down the operator
+**Note:** In the following commands, the affected `CephCluster` resource is called `rook-ceph`. If yours is named differently, the
```

Comment:
This will format the note nicely:
```suggestion
!!! note
    In the following commands, the affected `CephCluster` resource is called `rook-ceph`. If yours is named differently, the
```